src/main.rs

```diff
@@ -320,6 +343,8 @@ mod tests {
                 .expect("copying of OS source disk image failed");
             fs::copy(osdisk_raw_base_path, &osdisk_raw_path)
                 .expect("copying of OS source disk raw image failed");
+            fs::copy(cloudinit_path.clone(), &cloud_init_vfio_base_path)
```

Comment:
I think you can replace the .clone() with &

Reply:
Fixed, thanks.